### PR TITLE
style(rxApp): add style for disabled links

### LINKED ISSUE #715: Add disabled styles for links to the Framework
See: https://github.com/rackerlabs/encore-cloud-ui/pull/398/files#r22673865

### PR DESCRIPTION
**Pull request by glynnis**
Closes #715

Note that once Cloud upgrades its framework version after this has been released, there is clean-up to be done. See https://github.com/rackerlabs/encore-ui/issues/715 for details.

### CODE REVIEW

File: src/rxApp/rxApp.less [hunk 1]
```diff
@@ -2,102 +2,6 @@
...
-                            <p>Lorem ipsum dolor sit amet.</p>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </body>
-```
-
- */
```
Comment (Droogans) [inline src/rxApp/rxApp.less]: Is this out of date? In the template somewhere?

I'm wondering why this was included in the PR.

Comment (glynnis) [inline src/rxApp/rxApp.less]: It's documentation for StyleDocco, which we are no longer using, so I removed it.

**Comment by glynnis** [comment]
Der, just realized I need to update the demo to show this style too. Changed title to "DO NOT MERGE" until I get that in.

**Comment by comeatmebro** [comment]
[Wraith UI diff](http://eb4d4842ea70055541c8-b6a3ba129b5eb76af13f4d17e4c1a53b.r51.cf5.rackcdn.com/768/gallery.html)

**Comment by glynnis** [comment]
Added examples to Demo.

**Comment by comeatmebro** [comment]
[Wraith UI diff](http://eb4d4842ea70055541c8-b6a3ba129b5eb76af13f4d17e4c1a53b.r51.cf5.rackcdn.com/768/gallery.html)

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**